examples/plotting/map_editcolormap.py

```diff
@@ -17,12 +17,12 @@
 aiamap = sunpy.map.Map(AIA_171_IMAGE)
 
 ###############################################################################
-# All plot settings for a map are stored in the `plot_settings` attribute.
+# The colormap and normalization for a map are accessed as a property of the map.
 # How a Map is displayed is determined by its colormap, which sets the colors
 # , and the normalization, which sets how data values are translated to colors.
 # Lets replace the colormap and normalization.
-aiamap.plot_settings['cmap'] = plt.get_cmap('Greys_r')
-aiamap.plot_settings['norm'] = colors.LogNorm(100, aiamap.max())
+aiamap.cmap = plt.get_cmap('Greys_r')
```

Comment:
I am a little concerned that this example isn't really describing what we think is the best way of doing things. It should probably be completely re-written?